modules/nextflow/src/main/groovy/nextflow/executor/BridgeExecutor.groovy

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2020, Seqera Labs
```

Comment:
Your organization should be added in the copyright I guess

Reply:
Done !